src/lib/core/datetime.h

```diff
@@ -158,15 +158,33 @@ datetime_ev_now(struct datetime *now);
 void
 datetime_to_tm(const struct datetime *date, struct tnt_tm *tm);
 
+/**
+ * Create @sa datetime structure using given @sa tnt_tm fields.
+ */
+bool
+tm_to_datetime(struct tnt_tm *tm, struct datetime *date);
+
+/**
+ * Return whether given @sa datetime moment is DST.
+ */
+bool
+datetime_isdst(const struct datetime *date);
+
+/**
+ * Return gmtoff of a given @sa datetime moment.
+ */
+long
+datetime_gmtoff(const struct datetime *date);
+
 /**
  * Parse datetime text in ISO-8601 given format, and construct output
  * datetime value
  * @param date output datetime value
  * @param str input text in relaxed ISO-8601 format (0-terminated)
  * @param len length of str buffer
- * @param offset timezone offset, if you need to interpret
- *        date/time value as local. Pass 0 if you need to interpret it as UTC,
- *        or apply offset from string.
+ * @param tzsuffix timezone override, if you need to interpret
+ *        date/time value as local. Pass NULL if you need to interpret it
+ *        as UTC, or your provide meaningful suffix.
```

Comment:
I don't understand the last line

Reply:
I mean that there is already timezone suffix in the full datetime literal to be parsed - you do not need to pass tzsuffix here to get correctly understood by parser.

Reply:
Please, clarify it in comment:
`or you provide meaningful suffix in the literal to be parsed` or something similar.

Reply:
Fixed (but it's not force pushed yet - to wait till previous CI run completed to make sure it's ok).